[Docs]: Improve documentation about the installation of Gotenberg and LibreOffice (#15277)
pimcore/pimcore: doc/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/06_Additional_Tools_Installation.md

### Chromium (Chrome Headless)

> First of all, it would be required to add and install the required library via composer  

Reviewer: ```suggestion
First of all, you need to add and install the required library via composer:
```

```bash
composer require chrome-php/chrome
```

> It would be then possible to directly install and use the Chromium binary locally or in Docker Container and access to it via WebSocket connection 

Reviewer: ```suggestion
You can then directly install and use the Chromium binary locally or use it in Docker Container, accessing it via WebSocket connection.
```

#### Locally

> Please visit [https://www.chromium.org/getting-involved/download-chromium](https://www.chromium.org/getting-involved/download-chromium) and follow the instruction to install.

Reviewer: ```suggestion
Please follow the steps on [https://www.chromium.org/getting-involved/download-chromium](https://www.chromium.org/getting-involved/download-chromium) for installing Chromium locally.
```

#### Docker

> Add a new service as 

Reviewer: ```suggestion
Add a new service as:
```

```

> and set accordingly

Reviewer: ```suggestion
and set accordingly:
```

> - config `pimcore.chromium.uri` value , eg. `ws://chrome:3000/` 

Reviewer: ```suggestion
- config `pimcore.chromium.uri` value (e.g. `ws://chrome:3000/`) 
```

> - web2print settings hostUrl as the Docker web server service eg. `http://nginx:80`

Reviewer: ```suggestion
- web2print settings hostUrl as the Docker web server service (e.g. `http://nginx:80`)
```

- `pimcore.gotenberg.base_url` which by default to `gotenberg:3000`
- `pimcore.documents.preview_url_prefix` for example to `nginx:80`

> Make sure to add and install the required library via composer  

Reviewer: ```suggestion
Make sure to add and install the required library via composer:
```
